R/rows-db.R

```diff
@@ -292,11 +305,6 @@ rows_upsert.tbl_dbi <- function(x, y, by = NULL, ...,
   name <- target_table_name(x, in_place)
 
   if (!is_null(name)) {
-    # Checking optional, can rely on primary key constraint
-    if (is_true(check)) {
-      check_db_superset(x, y, by)
-    }
-
     if (is_empty(new_columns)) {
       return(invisible(x))
```

Comment:
Are we sure we want to return sometimes visibly sometimes invisibly ?

Reply:
For `in_place = TRUE` we do return invisibly, because this is an operation with side effects.